### PR TITLE
Add securityPolicy to gke-operator

### PR DESCRIPTION
**Pull request by thedadams**
The securityContext is necessary when adding additional CAs.

Issue:
https://github.com/rancher/rancher/issues/32903

### CODE REVIEW

**Review by cmurphy — DISMISSED** [review]
nit about the commit message, it's `securityContext`

**Comment by cmurphy** [comment]
This doesn't bring the chart into alignment with what's in gke-operator currently, do we need to add this too https://github.com/rancher/gke-operator/blob/6b98808efcf8067480818eecca2ceecfde242608/charts/gke-operator/templates/deployment.yaml#L39-L44 ?

**Comment by thedadams** [comment]
@cmurphy Good catches! Updated.

**Review by cmurphy — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]
LGTM

File: charts/rancher-gke-operator/1.1.200/Chart.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ description: A Helm chart for provisioning GKE clusters
 home: https://github.com/rancher/gke-operator
 sources:
   - "https://github.com/rancher/gke-operator"
-version: 1.0.400
```
Comment (jiaqiluo) [inline charts/rancher-gke-operator/1.1.200/Chart.yaml:7]: 1.0.400 is an unreleased version so we can overwrite the version.

File: charts/rancher-gke-operator/1.1.200/Chart.yaml [hunk 2]
```diff
@@ -4,5 +4,5 @@ description: A Helm chart for provisioning GKE clusters
...
-appVersion: v1.1.1-rc10
+version: 1.1.200
```
Comment (cbron) [inline charts/rancher-gke-operator/1.1.200/Chart.yaml:7]: Needs to be updated here: https://github.com/rancher/rancher/issues/33528

**Review by cbron — APPROVED** [review]
Lgtm. We should dev-validate that the basics here are working before sending to qa, meaning that the chart sets cert appropriately.

File: charts/rancher-gke-operator/1.1.200/Chart.yaml (hunk 2 above)
Comment (thedadams) [reply charts/rancher-gke-operator/1.1.200/Chart.yaml:7]: Updated version in linked issue.

**Pull request merged**